### ISSUE TITLE
Roslyn Analyzer for Razor Page when you have Tag Helper that takes input but no [BindProperty] attribute on Page Model

### ISSUE THREAD

**Issue by scottsauber** [aspnet/Mvc#7808]
#### Is this a Bug or Feature request?:
Feature Request

#### Description
Just a random idea I had, so I thought I'd log it.  It'd be cool if there was a Roslyn Analyzer for Razor Pages when you have a Tag Helper that takes input (such as input, select, etc.) and it references a property/object that doesn't have the [BindProperty] attribute.  Then it'd raise a diagnostic alerting you of this problem, ideally with a code fix as well.

I've hit a handful of times when creating new pages where I forget to add the [BindProperty] and it's not until I run it and do a POST and everything's null that I go "duh I forgot to add the attribute."  It may take someone who's new to Razor Pages a little while to figure that out, so this could be a time saver.

Is this something the team would be interested in?  If so, I wouldn't mind taking a crack at it.

**Comment by mkArtakMSFT** [comment]
As this is not a priority for the upcoming release, we'll park this issue in the Backlog for now, to observe community interest in this ask. @scottsauber

**Comment by mkArtakMSFT** [comment]
Hi. Thanks for contacting us.
We're closing this issue as there was no much community interest in this ask for quite a while now.
